amplify/backend/function/jcProcessDMAlerts/src/index.js

```diff
@@ -238,24 +288,57 @@ async function Execute(event) {
         const content = record.dynamodb.NewImage.content.S
         const from = record.dynamodb.NewImage.userId.S
         const recipients = record.dynamodb.NewImage.recipients.L.map((item) => item.S)
-        console.log(recipients)
+        let recipientsToBeNotified = recipients
```

Comment:
This is missing a check to see whether it is a DM or Course type.
It should skip this section if it is not Course type...
```
if(messageRoomID.startsWith("course")) { 
    
}